example-app/src/main/java/com/adyen/checkout/example/ui/compose/GenericDialog.kt

```diff
@@ -32,31 +32,28 @@ import com.adyen.checkout.example.ui.theme.ExampleTheme
 @Composable
 fun GenericDialog(
     modifier: Modifier = Modifier,
-    title: (@Composable () -> Unit)?,
-    content: (@Composable () -> Unit)?,
-    dismissButton: (@Composable () -> Unit)?,
-    confirmButton: (@Composable () -> Unit)?,
+    title: (@Composable () -> Unit)? = null,
+    content: (@Composable () -> Unit)? = null,
+    dismissButton: (@Composable () -> Unit)? = null,
+    confirmButton: (@Composable () -> Unit)? = null,
     onDismiss: () -> Unit,
 ) {
     Dialog(onDismiss) {
         Surface(
-            modifier = modifier.fillMaxWidth(),
             shape = MaterialTheme.shapes.medium,
             shadowElevation = 1.dp,
         ) {
-            Column {
-                Column(Modifier.padding(ExampleTheme.dimensions.grid_2)) {
-                    title?.invoke()
-                    if (title != null && content != null) {
-                        Spacer(Modifier.size(ExampleTheme.dimensions.grid_2))
-                    }
-                    content?.invoke()
+            Column(
+                modifier = modifier.fillMaxWidth(),
+            ) {
+                title?.invoke()
+                if (title != null && content != null) {
+                    Spacer(Modifier.size(ExampleTheme.dimensions.grid_2))
                 }
+                content?.invoke()
                 if (dismissButton != null || confirmButton != null) {
                     Row(
-                        Modifier
-                            .padding(ExampleTheme.dimensions.grid_1)
-                            .fillMaxWidth(),
+                        Modifier.fillMaxWidth(),
```

Comment:
The space between the buttons and the content has been removed

![Screenshot 2024-09-12 at 09 21 00](https://github.com/user-attachments/assets/22c49746-f1ab-467a-8f5d-2426d8fba2b2)

Reply:
Indeed, fixed it.